src/Pages/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
```

Comment:
1. design is not matching with Figma
2. make all fields mandatory
3. show customer error messages for email and other fields 
4. add validation for the name (name should be [a-zA-Z]) and email
5. use proper styling